website/src/components/props-table.tsx

```diff
@@ -42,19 +42,23 @@ const PropsTable = ({
     themeComponent?.variants && Object.keys(themeComponent.variants)
 
   /**
-   * If component has size prop, override the rendered values
-   * for `size` prop with  the component's size values
+   * If component has size prop, override the rendered value
+   * for `size` prop with the component's size values formatted as TS type.
    */
   if (info.props.size && sizeValues) {
-    info.props.size.type.name = sizeValues.join(", ")
+    info.props.size.type.name = sizeValues
```

Comment:
Hi @dodas, thanks for this, any reason why you are using the `.map` before the `.join`? What do you think about using this`:

```suggestion
    info.props.size.type.name = sizeValues.join(' | ')
```

Reply:
Hi @tioluwani94!

Without the `.map`, you would achieve following:
`ghost | outline | solid | link | unstyled`

instead of the following:
`"ghost" | "outline" | "solid" | "link" | "unstyled"`

I am trying to achieve a valid TS type here (same as `"horizontal" | "vertical"` in orientation).

Reply:
This makes sense. Thanks again @dodas